flink-ml-parent/flink-ml-api/src/test/java/org/apache/flink/ml/api/misc/ParamsTest.java

```diff
@@ -39,7 +40,18 @@ public void testDefaultBehavior() {
 
 		ParamInfo<String> optionalWithoutDefault =
 			ParamInfoFactory.createParamInfo("a", String.class).build();
-		assert params.get(optionalWithoutDefault) == null;
+		/**
+		 * It should call params.contain to check when get the param in this case.
+		 *
+		 * example:
+		 *
+		 * in {@link HasPredDetailColName}, it should be optional, and has not default value.
```

Comment:
I suppose we should not link from flink-ml-api model to a param defined in flink-ml-lib. flink-ml-api should be independent. Please check.

Reply:
removed